neuralpp/test/quick_tests/symoblic/simplication_test.py

```diff
@@ -22,33 +21,53 @@ def test_simplify() -> None:
     assert factor(x**2 + 2*x + 1) == (x + 1)**2
 
 
-def test_expand() -> None:
-    """ The function expand() expands. """
-    x = symbols('x')
+def test_polynomial_rational() -> None:
```

Comment:
The original name test_expand seemed clearer to me, I guess because "expand" is an action and we test actions. What does it mean to test a polynomial_rational?

Reply:
A better name may be `test_polynomial_or_rational_function_simplification`. The library has a dedicated set of simplification functions for each kind of expressions, e.g., `expand()` and `factor()` for polynomial/rational, `trigsimp()` for trigonometric.

Reply:
Can you please update the name then? Thanks.

Reply:
Changed the function names in this file.